grpc.go

```diff
@@ -439,6 +342,18 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
+	for idx, mnt := range config.Mounts {
+		if mnt.Destination == "/dev/shm" {
+			config.Mounts[idx] = &configs.Mount{
+				Device:      "tmpfs",
+				Source:      "shm",
+				Destination: "/dev/shm",
+				Data:        "mode=1777,size=65536k",
+				Flags:       defaultMountFlags,
+			}
+		}
+	}
+
```

Comment:
> The specific case of /dev/shm requires the agent to handle this mount
in a special way that cannot be properly handled by the runtime.

I don't get it. Why the runtime can't pass the config as the above code?

Reply:
@sboeuf I agree with @laijs: This mount should be curated from the runtime.

Reply:
Removed! I am gonna try to make this work another way.